include/core/newton_non_linear_solver.h

```diff
@@ -108,7 +108,7 @@ NewtonNonLinearSolver<VectorType>::solve(
 
       solver->solve_linear_system(first_step);
 
-      for (double alpha = 1.0; alpha > 1e-3; alpha *= 0.5)
+      for (double alpha = 1.0; alpha > 1e-1; alpha *= 0.5)
```

Comment:
should this threshold tolerance be defined in the input prm file?

Reply:
No I don't think it should. This is just too specific it would add a useless parameters. It was nonsensical to have it so low anyway.